crud/select/plan.lua

```diff
@@ -53,9 +53,10 @@ local function validate_conditions(conditions, space_indexes, space_format)
     return true
 end
 
+
```

Comment:
Usually we keep only one line between functions. Why do you add this empty line?